nemo/collections/multimodal/data/audio_text_qa_dataset.py

```diff
@@ -923,6 +923,8 @@ def get_tarred_aqa_dataset(
         else:
             datasets.append(dataset)
 
+    with open_dict(config):  # patch for bucketing tarred datasets
+        config['batch_size'] = config.get("micro_batch_size", 1)
```

Comment:
`config['batch_size']` is used by `get_chain_dataset()` for printing only, not actually used for dataloader